db/migrate/20230420114025_add_unique_index_to_users_email.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueIndexToUsersEmail < ActiveRecord::Migration[7.0]
+  def change
+    add_index :users, :email, unique: true, where: "email IS NOT NULL AND email != ''"
+  end
+end
```

Comment:
- [ ] This is actually causing an error during migration kindly remove as email is already unique in the previous migration file